shesha/scripts/saxoPlusScriptExample.py

```diff
@@ -47,22 +47,19 @@
 saxo.wfs.set_noise(0, -1) #removing noise on SH WFS (-1 = no noise)
 saxo.rtc.reset_ref_slopes(0) # reset refslopes from saxo to 0 (check)                    
 saxoplusmanager.iterations=0 # reset iterations count
-saxo.reset_wfs_exposure()
```

Comment:
Are you sure it has to be removed ?

Reply:
Yes. I have checked that it now brings the same results with and without reset_wfs_exposure.
This is because the call to next() stageSupervisor has now changed, and we can directly act on the stacking or not stacking of the WFS through a keyword.